workspace/src/main/openapi/workspace.yaml

```diff
@@ -882,34 +970,49 @@ components:
           type: object
           description: a map of RunTemplateId/DatasetId to set a default dataset for a Run Template
           additionalProperties: true
-      # required:
-        # - solutionId
+      required:
+      - solutionId
+
+    # List Response Schemas
+    WorkspacePermissionsList:
+      type: array
+      description: List of permissions for a workspace role
+      items:
+        type: string
+      example:
+        - "read"
+        - "read_security"
+        - "create_children"
+
+    WorkspaceUsersList:
+      type: array
+      description: List of users with access to the workspace
+      items:
+        type: string
+      example:
+        - alice@mycompany.com
+        - bob@mycompany.com
   examples:
     BreweryWorkspace:
       summary: Brewery Workspace
       description: Brewery Workspace example
       value:
-        id: "w-da345dia76fg"
-        organisationId: "o-daup4fea311"
         key: brewery
         name: Brewery Analysis
         description: Brewery analysis for stock, production, transport and customer satisfaction
-        linkedDatasetIdList:
-          - "d-pe57ai0h34"
         version: "1.0.0"
         tags:
           - Brewery
-        ownerId: "da3518zpo-coizou3154851354dam-izhva38fze1"
         solution:
           solutionId: "sol-feo16eua48o"
           runTemplateFilter:
             - "run1"
           defaultRunTemplateDataset:
             run1: "data1"
-        sendInputToDataWarehouse: null
-        useDedicatekdEventHubNamespace: false
-        dedicatedEventHubSasKeyName: null
-        dedicatedEventHubAuthenticationStrategy: null
+        sendInputToDataWarehouse: false
+        useDedicatedEventHubNamespace: false
+        dedicatedEventHubSasKeyName: "RootManageSharedAccessKey"
+        dedicatedEventHubAuthenticationStrategy: "SHARED_ACCESS_POLICY"
```

Comment:
Did you do an analysis of the fields we can remove or is it scheduled for later? For example I don't think these fields (and others in the Workspace object) are used/useful anymore with the new orchestrator. There is a huge cleanup to do on Workspace in particular if I remember.